apiserver/pkg/model/model.go

```diff
@@ -304,6 +308,11 @@ func ModelFiles(ctx context.Context, c dynamic.Interface, modelName, namespace s
 	}
 	objectInfoList := anyObjectInfoList.([]miniogo.ObjectInfo)
 	sort.Slice(objectInfoList, func(i, j int) bool {
+		// make the readme file top.
+		isReadme := strings.ToLower(objectInfoList[i].Key)
+		if strings.HasSuffix(isReadme, readmeLower) {
```

Comment:
We can return this file name to front-end , so they can do extrac file query.

Reply:
besides... do we need to provide another api to view README online ?

Reply:
This pr can be closed since we should let front-end list files and filter the `README` file